packages/react/src/table/TableRow.tsx

```diff
@@ -1,13 +1,15 @@
 import { PropsWithChildren } from 'react';
-import { boxPalette, tokens } from '../core';
 import { theme } from '../ag-branding';
+import { boxPalette, tokens } from '../core';
 import { useTableContext } from './TableContext';
 
 export type TableRowProps = PropsWithChildren<{
 	/** The row index of the table row. */
 	'aria-rowindex'?: number;
 	/** Style the row when a cell contains an error. */
 	invalid?: boolean;
+	/** Callback function to execute when the row is clicked. Callback is not run when the clicked element is not a th, tr or td. */
```

Comment:
Not quite truthful, but consumers don't need to know the details. Just alphabetise
```suggestion
	/** Callback function to execute when the row is clicked. Callback is not run when the clicked element is not a td, th, or tr. */
```

Reply:
Updated in [8e975b](https://github.com/agriculturegovau/agds-next/pull/1994/commits/8e975b987c9ea08935a97ce064686629d4fa26da)